res/enkf/enkf_main.py

```diff
@@ -376,8 +375,8 @@ def loadFromRunContext(self, run_context, fs):
     def initRun(self, run_context):
         self._init_run(run_context)
 
-    def createRunpath(self, run_context):
-        self._create_run_path(run_context)
+    def createRunPath(self, run_context: ErtRunContext) -> None:
```

Comment:
Would it be possible to delete this instead of having it duplicated? It does just call this one anyway so not a big deal, but would simplify the code somewhat.

Reply:
Do you mean delete `createRunPath` from `enkf_simulation_runner.py` which just calls `self.parent().createRunPath()` which is the function implemented in `enkf_main.py`?

Reply:
Yes. or visa versa, whichever is easier.

Reply:
Ended up removing it from `enkf_main.py`, thanks 👍